{{cookiecutter.project_name}}/.mlops-setup-scripts/cicd/common/main.tf

```diff
@@ -19,10 +19,16 @@ resource "databricks_secret" "cd_credentials_token" {
   scope        = databricks_secret_scope.cd_credentials.id
 }
 
+resource "databricks_secret" "cd_github_server" {
+  key = "github_server"
+  string_value = var.github_server_url
+  scope = databricks_secret_scope.cd_credentials.id
+}
+
 resource "databricks_secret" "cd_github_repo" {
```

Comment:
Same thing here - why is the github_repo a databricks_secret?

Reply:
Please see above and correct me if I explained it wrong @arpitjasa-db